### ISSUE TITLE
freeze on "freely transformable windows"

### ISSUE THREAD

**Issue by fraterchaos** [compiz-reloaded/compiz#6]
When I activate the transformable windows feature, and rotate of transform a window, it often freezes the windows. Sometimes they go dark and refuse to move or revert, other times they will return to normal color and brightness on their own.

Often, it will not return, the windows will stay frozen. Sometimes I can switch to another TTY and switch back, and then use the reset key combo to restore the windows, but most times once they've frozen the only answer is to restart the computer.

I am running Compiz-reloaded under Manjaro Linux version 15.12 Capella, with the 4.5-2 kernel. Running on an Asus buldozer Motherboard with an AMD FX 8350 octacore processor, ATI Raedeon Sapphire 7770 HD graphics. 16 Gigs on board RAM.

**Comment by soreau** [comment]
Yes, there are known issues with freewins, it's possibly the buggiest plugin in the mix. Patches welcome.
